profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.info

```diff
@@ -8,3 +8,4 @@ dependencies[] = nexteuropa_token
 dependencies[] = wysiwyg
 dependencies[] = views
 files[] = views/nexteuropa_token_ckeditor_handler_field_selection.inc
+files[] = views/nexteuropa_token_ckeditor_bean_handler_field_selection.inc
```

Comment:
@drupol  Could you please update the documentation on github manuals to reflect the current changes? This is a requirement in normal project code reviews.

I would be grateful if some developer guidelines on API usage is mentioned.